aaelf64-morello/aaelf64-morello.rst

```diff
@@ -1,13 +1,13 @@
 ..
-   Copyright (c) 2020, Arm Limited and its affiliates.  All rights reserved.
```

Comment:
when there are only two years, it should be `2020, 2021`

Reply:
Correct.